src/core/batch/batch.test.ts

```diff
@@ -1,5 +1,13 @@
-import { batchIncludes } from "./batch";
-import { ParquetReader } from "@dsnp/parquetjs";
+import * as batch from "./batch";
+import { generateBroadcast } from "../../generators/dsnpGenerators";
+import { EmptyArrayError } from "../utilities/errors";
+import { putStream } from "../store/interface";
+import { BroadcastSchema } from "./parquetSchema";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const parquet = require("@dsnp/parquetjs");
+const { ParquetReader, ParquetWriter } = parquet;
```

Comment:
This should work now
```suggestion
import { ParquetReader, ParquetWriter } from "@dsnp/parquetjs";
```